internal/provider/kubernetes/status_test.go

```diff
@@ -361,24 +361,6 @@ func Test_mergeRouteParentStatus(t *testing.T) {
 						},
 					},
 				},
-				{
-					ControllerName: "gateway.envoyproxy.io/gatewayclass-controller",
```

Comment:
@arkodg This status has been removed because this PR merge the status across multiple translation and update it only once at the end with the combined result.